src/render.js

```diff
@@ -0,0 +1,46 @@
+const updateLocalStorage = require('./storage.js');
+
+function renderTasks(tasks) {
+  const list = document.getElementById('todoList');
+  list.innerHTML = '';
+  tasks.forEach((item) => {
+    const li = document.createElement('li');
+    li.setAttribute('class', 'list-name');
+    if (item.completed) {
+      li.classList.add('checked');
+    } else {
+      li.classList.remove('checked');
+    }
+    const checked = item.completed ? 'checked' : '';
+
+    li.innerHTML = `
+            <input type="checkbox" class="checkbox" id=${item.index} ${checked}>
+            <span>${item.description}</span>
+            <button class="remove"></button>
+          `;
+    list.append(li);
+  });
+  //   const list = document.getElementById('todoList');
```

Comment:
- [ ]  Kindly remove all commented codes in your files. It makes your code unreadable and a little messy.  :pray:

Reply:
This issue has been well addressed. Great job! :pray: